norlen/d0018e-webshop
# Create order from cart

First draft, will close #4 later.

Still todo:
- page for order done
- testing
- may want to make sure the price matches (what the client sees, and our values from the database) thoughts?

however, check the styling (changed bg to a gray color) and updated the products slightly. wanted a box for some stuff on the checkout page and a different bg color seemed good then.

the way the checkout works is that
1. Send customer info + all prodcuts (id + quantity) you want to buy. We could check the DB for this, but it seemed better to pass along what the client actually sees, rather than something from the DB (for example, if products were added from another tab)
2. Run a SQL transaction (create order, iterate over items and create order_products, then remove all items from cart)
